pkgs/applications/science/molecular-dynamics/hoomd-blue/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchgit
+, cmake, pkgconfig
+, python, numpy
```

Comment:
Don't pass in attributes of the Python package set. Instead pass in `pythonPackages` and use its attributes, or pass in `python`, and use `python.pkgs`' attributes.